### ISSUE TITLE
[Bug]: Chunks load very slowly when the mod is installed

### ISSUE THREAD

**Issue by Raidobw2** [ThePaul-T/WYML#24]
Hello Paul,

I came accross an issue with WYML installed where chunks would generate/load very slowly. I was using a dev install of Plexiglass Mountain with a full updated modlist, stripping mods in batches of 20 to find the issue.

WYML miraculously made it to the end, with around 15 mods remaining after removing a lot of them. When I took out WYML, I noticed a significant improvement in chunk loading/generation.

All important visual information is on ticket https://github.com/FTBTeam/FTB-Modpack-Issues/issues/1245 where I compare with/without WYML installed, with videos.

If you check the "tip" of where the chunks currently are loading/generating, you'll notice the difference right away.

Is this something WYML could influence and would it be fixable? I've attached spark reports from the videos to help further.
Having chunk generating at full speed would help the modpack's experience a lot, and this might affect the DW20 1.18 modpack too.

Let me know if I can be of any help!! Maybe to help you test builds if you need a helping hand

Thanks in advance :D

**Comment by Raidobw2** [comment]
Thanks for the new config option in 1.1.4!

I've tried it and I'm sure it helps with chunk gen, though I can't really confirm it... See below, you'll understand why :o

I'm using the new config `"DISABLE_COUNTING_CHUNK_GENERATED_MOBS": true`

This is without WYML, chunk loading speed:

https://user-images.githubusercontent.com/17010632/186606206-e4f3e530-fa7c-4384-bb90-6ac18983fda7.mp4

This is with WYML:

https://user-images.githubusercontent.com/17010632/186606275-ccce1670-f2e7-4fc9-b6dd-ab2ebbd358a4.mp4

I'd like to say chunk gen speed improved... But since visual chunk load speed looks currently slower than visual chunk gen speed, it's difficult for me to tell visually. All those chunks were already generated. If you'd like me to test more things, let me know and thanks for your work! :)

**Comment by ThePaul-T** [comment]
Thanks @Raidobw2, we'll look further, the visual aid is helpful. We commented out the entire chunk-gen part of the code and couldn't see any real improvement in speed, so we was struggling to move forward. 

Will have another look with @gigabit101 today, the new changes with entity storage in Minecraft are making this difficult for us to adapt to.

**Comment by Raidobw2** [comment]
I've tested again earlier, and here are my findings! As you suspected, the NORMALIZE_TICKS option in the configuration slows down loading chunks.

This is when WYML is installed, and NORMALIZE_TICKS is false:

https://user-images.githubusercontent.com/17010632/186829841-10969b0a-6a6a-43eb-bb84-e5a530ad3452.mp4

-----

I've also went and checked for any world generation improvements within WYML, here is what world generation looks like when the mod is NOT installed:

https://user-images.githubusercontent.com/17010632/186830020-cce9d55d-2848-4020-b90e-22e65b39e0b8.mp4

And finally, when WYML is installed and the new worldgen option (DISABLE_COUNTING_CHUNK_GENERATED_MOBS) is set to true:

https://user-images.githubusercontent.com/17010632/186830194-71ce986c-6725-48a2-9a94-a3633bdadf63.mp4

I didn't check back what the chunks generating looked back then, but at least I see now that world generation with disable counting chunk generated mobs is the same speed as normal "vanilla" worldgen -- well my eyes don't catch any differences.

And now what the mystery around chunks loading slowly is solved, I believe this issue is then fixed simply by ticking off the NORMALIZE_TICKS field in the config.

I'll assume there is nothing else to do, using the configuration satisfies me. Thanks a lot for your help and gigabit101's help!
